errors/go.mod

```diff
@@ -0,0 +1,11 @@
+module errors
```

Comment:
What happened to the `core`/`base` idea discussed in #9011?

in my mind `errors` has too small scope to warrant its own module. Same applies to `math` or `codec`. I personally prefer to have one `core` module to regroup these small building blocks, instead of having to maintain 10+ small go modules.

Reply:
I tend to sort of agree here. What are your thoughts @aaronc?

Reply:
I'm going to write up my thoughts in a discussion thread and link back here